notesserver/highlight.py

```diff
@@ -39,13 +39,28 @@ def build_search_kwargs(self, *args, **kwargs):
             highlight_options = {
                 'fields': {
                     content_field: {'store': 'yes'},
-                }
+                    'tags': {'store': 'yes'},
+                },
             }
             if isinstance(highlight, dict):
                 highlight_options.update(highlight)
             res['highlight'] = highlight_options
         return res
 
+    def _process_results(
```

Comment:
to clarify, i only meant to propose highlighting tags on the client, not text in the body of the document.  this is based on some assumptions i'm making, for example that you're analyzing / matching tags as keywords, rather than doing any tokenization / stemming there.  assuming that, frontend code could check tags as they're being rendered and do a simple equality check to decide whether they should be styled specially (highlighted) or not.  another assumption i'm making is that the frontend is built in such a way that this is feasible (e.g. you have easy access to the text that was searched at the moment you are rendering the tags in the results).